packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -49,7 +49,8 @@ export class TerminalFrontendContribution implements CommandContribution, MenuCo
 
     registerMenus(menus: MenuModelRegistry): void {
         menus.registerMenuAction(CommonMenus.FILE_NEW, {
-            commandId: TerminalCommands.NEW.id
+            commandId: TerminalCommands.NEW.id,
+            label: 'Open New Terminal'
```

Comment:
I'd prefer this for the command, as well. This entry is rendered in the global menu anyways, so there is not more context than in the command palette.